webpack.config.js

```diff
@@ -139,9 +143,16 @@ module.exports = {
       }
     ],
   },
-  plugins: [],
+  plugins: [
+    // Strip *all* locales from Moment.js.
+    new MomentLocalesPlugin()
+  ],
 };
 
+if (shouldAnalyze) {
+  module.exports.plugins.push(new BundleAnalyzerPlugin());
```

Comment:
Should we add a comment here or in the `README` that says that the build will fail if the bundle size increases too much? Not sure if a new person would be comfortable enough with CI to be able to debug that if the PR fails the check/would think to look at the `.yml` file.

Reply:
Maybe related to the concept of adding external dependencies.

Reply:
FWIW, this line doesn’t have anything to do with the test that checks the size. But that’s reason enough that I should add a comment to explain it. :)

FWIW, I set up the size test to run as a separate step with a descriptive name — I was thinking that would be clear enough as to what’s happening:

![Screen Shot 2020-02-11 at 4 21 01 PM](https://user-images.githubusercontent.com/74178/74291709-9ab78c80-4cea-11ea-9e48-99b29153636d.png)

Otherwise we should probably just use BundleWatch or something more complicated so it’s really clear without having to add documentation. I don’t think this is something people should have to read or know about to work on the project.